pkgs/data/fonts/cabin/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "cabin-1.005";
+
+  src = fetchFromGitHub {
+    owner = "impallari";
+    repo = "Cabin";
+    rev = "982839c790e9dc57c343972aa34c51ed3b3677fd";
+    name = "${name}-src";
+    sha256 = "16v7spviphvdh2rrr8klv11lc9hxphg12ddf0qs7xdx801ri0ppn";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/opentype
+    mkdir -p $out/share/doc/${name}
+    cp -v "fonts/OTF/"*.otf $out/share/fonts/opentype/
+    cp -v README.md FONTLOG.txt $out/share/doc/${name}
+  '';
+
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "1ax5c2iab48qsk9zn3gjvqaib2lnlm25f1wr0aysf5ngw0y0jkrd";
```

Comment:
Ah, I guess you copied it over from the google fonts derivation? I think it makes sense for the google fonts since that package is so large, these ones are much smaller so it is not so critical here.